content/en/synthetics/settings/_index.md

```diff
@@ -57,7 +57,7 @@ You can create variables from your existing [HTTP tests][1] by parsing their ass
 5. Enable obfuscation of your variable to hide its value on test results (optional).
 6. Decide whether to extract your variable from the response headers or from the response body.
     * Extract the value from **Response Header**: use the full response header for your variable or parse it with a [regex][2].
-    * Extract the value from **Response Body**: parse the response body of the request with a [jsonpath][3], a [regex][2], or use the full response body.
+    * Extract the value from **Response Body**: parse the response body of the request with a [regex][2], a [`jsonpath`][3], an [`xpath`][4], or use the full response body.
```

Comment:
Open question
Why are JSONPATH and XPATH having a grey background?

Reply:
I'm following the same monospace format for JSONPath and XPath we use for the other doc pages.